src/App.jsx

```diff
@@ -11,7 +11,7 @@ function App() {
         <h1>Countries Stats</h1>
       </header>
       <Routes>
-        <Route path="/" element={<MainPage />} />
+        <Route path="/countries-database/" element={<MainPage />} />
```

Comment:
- [ ] Please kindly recheck the project and ensure it works as expected 
- [ ] According to the project requirement 1. The project contains a `Home` page and a `Detail` page.
1. The Home page has a list of categories that could be filtered using given parameters, e.g. category name.
1. It's possible to navigate the detail page by clicking the individual category item, there's a link from the `Detail` page to the `Home` page by clicking the "Back" button. please fix this and run it well to be sure it work as expected .